### PR TITLE
DMS section int the how to docs

### PR DESCRIPTION
**Pull request by AwkwardBen**

### CODE REVIEW

File: how-to-docs/dms-migration.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+### **Prerequisite**
+
+The following need to be true before you follow this guide:
+* AWS console logon
+* Access to the DMS service from console
+* A region where STS has been activated
+
+### **DMS Setup** 
+
+Login to the AWS console using your auth, switch to a role with the correct access policies and verify your in the right region. Next, slect DMS from the services on the main dashboard to access to data migration home screen. Under the "Get started" section click on the "create migration" button then next to the Replication instance. You should see the following screen:
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: `verify your` you're. `slect`, ` access to data migration` access the,

File: how-to-docs/dms-migration.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+![Alt text](pics/dms-doc-1.png?raw=true)
+
+The following are the options and example answer for the replication instance:
+
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: an example

File: how-to-docs/dms-migration.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+VPC | vpc-* | The virtual private cloud resource where you wish to add your dms instance. This should be as close to both the source and target instance as possible.
+Multi-AZ | No | Optional parameter to create a standby replica of your replication instance in another Availability Zone. Used for failover.
+Publicly Accessible | False | Option to access your instance from the internet
+
+You won't need to set any of the advanced settings. To create the instance click on the next button. You should now see a screen like this: 
+
+![Alt text](pics/dms-doc-2.png?raw=true)
+
+The following are the options and example answer for this endpoints instances page:
+
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: an example

File: how-to-docs/dms-migration.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+Source/target engine | postgres | Choose the type of database engine that for this endpoint.
+Server name | mysqlsrvinst.abcd123456789.us-west-1.rds.amazonaws.com | Type of server name. For an on-premises database, this can be the IP address or the public hostname. For an Amazon RDS DB instance, this can be the endpoint for the DB instance.
+Port | 5432 | The port used by the database.
+SSL mode | None | SSL mode for encryption for your endpoints. 
+Username | root | The user name with the permissions required to allow data migration.
+Password | ******** | The password for the account with the required permissions.
+Database Name (target) | dev-db | The name of the attached database to the selcted endpoint.
+
+Repeat these options for both source and target and make sure to test connection before clicking next. You might need to append rules the RDS roles to allow the replication instance access. Once this has fully been setup click next and you should be able to view the tasks page:
+
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: test the connection, `You might need to append rules the RDS roles` what does this mean?

File: how-to-docs/dms-migration.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+# **DMS Migration**
+
+### **Prerequisite**
+
+The following need to be true before you follow this guide:
+* AWS console logon
+* Access the the DMS service from console
+* A region where STS has been activated
```
Comment (Joseph-Irving) [inline how-to-docs/dms-migration.md]: the the

Comment (AwkwardBen) [inline how-to-docs/dms-migration.md]: haha the the DMS ^^

File: how-to-docs/dms-migration.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+Option | Example answer | Description
+------ | -------------- | -----------
+Endpoint identifer | database-source/target | This is the name you want to use to identify the endpoint.
+Source/target engine | postgres | Choose the type of database engine that for this endpoint.
+Server name | mysqlsrvinst.abcd123456789.us-west-1.rds.amazonaws.com | Type of server name. For an on-premises database, this can be the IP address or the public hostname. For an Amazon RDS DB instance, this can be the endpoint for the DB instance.
+Port | 5432 | The port used by the database.
+SSL mode | None | SSL mode for encryption for your endpoints. 
+Username | root | The user name with the permissions required to allow data migration.
+Password | ******** | The password for the account with the required permissions.
+Database Name (target) | dev-db | The name of the attached database to the selcted endpoint.
```
Comment (oyelekci) [inline how-to-docs/dms-migration.md]: typo - 'selected' endpoint

File: how-to-docs/dms-migration.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+The following are the options and an example of answers for this tasks page:
+Option | Example answer | Description
+------ | -------------- | -----------
+Task name | Migration-task | A name for the task.
+Task Description | Task for migrating | A description for the task.
+Source endpoint | source-instance | The source endpoint for migration.
+Target endpoint | target-instance | The target endpoint for migration.
+Replication instance | replication-instance | The replication isntance which will be used.
+Migration type | Migrate existing data | Migration method you wnat to use.
```
Comment (oyelekci) [inline how-to-docs/dms-migration.md]: typo - 'want to use'

File: how-to-docs/dms-migration.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+![Alt text](pics/dms-doc-4.png?raw=true)
+
+The following are the options and an example of answers for this tasks page:
+Option | Example answer | Description
+------ | -------------- | -----------
+Task name | Migration-task | A name for the task.
+Task Description | Task for migrating | A description for the task.
+Source endpoint | source-instance | The source endpoint for migration.
+Target endpoint | target-instance | The target endpoint for migration.
+Replication instance | replication-instance | The replication isntance which will be used.
```
Comment (oyelekci) [inline how-to-docs/dms-migration.md]: typo - 'instance' (second occurance)

File: how-to-docs/dms-migration.md (hunk 2 above)
Comment (AwkwardBen) [inline how-to-docs/dms-migration.md]: Changed

**Comment by oyelekci** [comment]
LGTM

**Pull request closed without merging**